earth_enterprise/docker/README.md

```diff
@@ -16,55 +16,53 @@ Add users that you want to have access to Docker to the `docker` group:
 # adduser muser docker
 ```
 
-### Cent OS 7
+### CentOS 7
 
 Install Docker as root:
 
 ```BASH
 yum install docker
-# In case your Docker deamon isn't running:
+
+# in case your Docker deamon isn't running:
 service docker start
 ```
 
 Now you can use Docker as the root user.
 
+## Building Images
 
-## Building the Images
-
-Before building Open GEE Docker images, make sure you have, at least, about
-8 GB of free disk space.
+Before building Open GEE Docker images, make sure you have at least 8 GB of
+free disk space.
 
 ### Image Configuration
 
-There are a number of option of how you can build and run Open GEE Docker
-images: 
+There are a number of options for building and running Open GEE Docker
+images:
 
-    * Platform to build on,
-    * Source code origin: from the main repository, or a fork, from the
-      master, or another branch, or from the current repository clone on your
-      disk;
-    * Port to forward HTTP connections to;
-    * Custom image to run, e.g., containing extra data like tutorial
-      resources, or custom built globes.
+* Build platform
+* Source code origin: from the main repository, a fork, another branch,
```

Comment:
I think this one became less clear than the original. You could be cloning from `master` on the main repository, another branch on the main repository, `master` on a fork, another branch on a fork, or use current files on your disk.